utah-design-system-website/src/@types/jsDocTypes.d.js

```diff
@@ -380,6 +380,12 @@
  * }
  */
 
+/**
+ * @typedef VerticalMenuExamplePropsShape {
+ *  @property {'inline' | 'flyout' | 'plain'} childrenMenuType
```

Comment:
mega menu is intentionally not an option? maybe a comment here about why it's not an option would be helpful.